### ISSUE TITLE
Wrong track duration for stereo .mp3 files

### ISSUE THREAD

**Issue by Ellusive1** [Zeugma440/atldotnet#211]
## The problem

When creating a `Track` object for a stereo mp3 file, the `Duration` and `DurationMs` properties are twice what they should be.

## Environment

* ATL version: z440.atl.core 4.34.0

## Details

The problem does not arise for mono files or stereo files in .wav / .ogg / .aac / joint stereo .mp3.
Only for .mp3 files with `ChannelsArrangement.Description` equals to `Stereo (2/0.0)`.

## Code To Reproduce Issue

```csharp
var = File.ReadAllBytes("./sample-6s.mp3");

using (MemoryStream memoryStream = new MemoryStream(mp3Bytes))
{
    var track = new Track( memoryStream );

    Console.WriteLine(track.DurationMs)// display 12872, should be  6436
}
```

Sample audio file: https://file.io/mVQpFtw6QGY9

**Comment by Zeugma440** [comment]
Thanks for your feedback. I'll get back to you next week-end.

Unfortunately I can't do anything right now as I'm on vacation 😉

**Comment by Zeugma440** [comment]
The issue does not happen with all Stereo MP3s, hopefully 😅 

The file you submitted has a very peculiar structure, as its MPEG headers contradict each other :
- The 1st header at offset 45, declares a 64 kbps stereo MP3 stream => the information ATL detects
- The 2nd header at offset 253 and all the others after that one, declare a 128 kbps joint stereo MP3 stream => the information that is supposed to be detected

I've adjusted the code for reading MPEG headers to be able to detect such cases. Now it's slightly slower but it covers your issue ;)

**Comment by Zeugma440** [comment]
Fix is available on today's v5.00
